pkg/db/sqlc/identity_updates.sql

```diff
@@ -35,4 +35,9 @@ SET
 	revocation_sequence_id = @revocation_sequence_id
 WHERE
 	address = @address
-	AND inbox_id = decode(@inbox_id, 'hex');
\ No newline at end of file
+	AND inbox_id = decode(@inbox_id, 'hex');
+
+-- name: AdvisoryLockSequence :exec
```

Comment:
Might think about a more specific name, since we could end up with more "advisory lock sequence"s for different functions.